Makefile

```diff
@@ -130,6 +130,21 @@ docker-test-integration: .coverage
 		-v $(CURDIR):/usr/src/github.com/vmware-tanzu/antrea:ro \
 		antrea/test test-integration $(USERID) $(GRPID)
 
+.PHONY: docker-test-integration-registry
+docker-test-integration: .coverage
```

Comment:
wrong target name, this is the same name as the target above

I don't like duplicating the target like this, do you think we can disable the `--pull` in docker commands by using an environment variable. `NO_DOCKER_PULL=true make docker-test-integration`?

Reply:
Indeed. Use `if [[ DOCKER_REGISTRY == "" ]]` as judgement in `Makefile`

Reply:
Good idea!